front/app/components/InitiativeForm/index.tsx

```diff
@@ -36,6 +36,7 @@ const Form = styled.form`
   display: flex;
   flex-direction: column;
   margin-bottom: 100px;
+  align-items: center;
```

Comment:
Centers the form elements in /admin/initiatives/:initiativeId with no adverse effects (it seems) on other places where this component is used.